Assets/Scripts/Game/Game.cs

```diff
@@ -28,7 +30,9 @@ public void PlaceTile(int tileID, int q, int r, int marketIndex = 0)
 		int marketAdjustment = marketIndex > 1 ? (marketIndex - 1) * 2 : 0;
 		if (activePlayer.money < tile.price + marketAdjustment)
 		{
-			PopupManager.Show ("Not enough money.");
+			Debug.Log (pack.GetMsgLocalization()[4].message);
```

Comment:
Get rid of this debug message. Calling `Debug.Log` should be reserved for debugging information.